jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-disruptive-ha-mariadb-template.yaml

```diff
@@ -0,0 +1,35 @@
+- job-template:
+    ### The reason why this upgrade job is "disruptive" is the default cinder backend
+    name: 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-mariadb-{arch}'
+    node: cloud-trigger
+
+    triggers:
+      - timed: '32 22 * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{previous_version}
+            upgrade_cloudsource=develcloud{version}
+            ### 3 nodes for HA cluster, 2 compute nodes, 1 lonelynode for nfs server
+            nodenumber=6
```

Comment:
Is this correct, or do I need to set `nodenumber=5` and `nodenumberlonelynode=1` ?